packages/simulator/lib/fleet.js

```diff
@@ -39,13 +40,17 @@ class Fleet {
   constructor({ name, marketshare, percentageHomeDelivery, vehicles, hub }) {
     this.name = name
     this.marketshare = marketshare
-    this.hub = { position: convertPosition(hub) }
+    this.hub = {
+      position: hub.length
+        ? convertPosition(hub)
+        : pelias.search(hub).then((position) => (this.hub = position)),
```

Comment:
Denna är till för att man ska kunna skriva in en adress istället i fleets.json - borde göras lite snyggare